frontend/index.html

```diff
@@ -1,5 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
+<head>
+  <meta charset="UTF-8">
+</head>
```

Comment:
This already exists in the production site - it's probably added dynamically. Are you sure it should be added here?

Reply:
Good question. I was relying on the old cypress tests which seemed to want that change at least in my local dev environment. I'll have a look and see if I can find how it is added in the production site. (Could be that the test is just old and needs to be removed I guess)

Reply:
This was a mistake. I have reverted the change and updated the test to work with the current site.

Reply:
Thanks for fixing it!